uniffi_bindgen/src/scaffolding.rs

```diff
@@ -78,7 +78,7 @@ mod filters {
         // implementations of the functions that we're wrapping (and also to type-check our generated code).
         // This will panic if the bindings provide an invalid value over the FFI.
         Ok(format!(
-            "<{} as uniffi::ViaFfi>::try_from_ffi_value({}).unwrap()",
```

Comment:
If y'all hate "lift" and "lower" in practice, we could also change this to use the more verbose "into_ffi_value" and "from_ffi_value" names everywhere, rather than getting rid of them on the rust side.